frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -142,7 +142,7 @@ export const dashboardLogic = kea<dashboardLogicType>([
     }),
 
     actions({
-        loadDashboardItems: (payload: { refresh?: boolean; action: string }) => payload,
```

Comment:
This was some _really_ legacy naming, from back when this literally had to load every dashboard item individually